src/main/java/uk/gov/pay/api/model/directdebit/mandates/GetMandateResponse.java

```diff
@@ -7,74 +7,77 @@
 
 @ApiModel(value = "GetMandateResponse")
 public class GetMandateResponse {
-
-    private static final String AGREEMENT_ID_FIELD_NAME = "agreement_id";
-    private static final String RETURN_URL_FIELD_NAME = "return_url";
-    private static final String STATE_FIELD_NAME = "state";
-    private static final String LINKS_FIELD_NAME = "_links";
-    private static final String PROVIDER_ID_FIELD_NAME = "provider_id";
-    private static final String REFERENCE_FIELD_NAME = "reference";
-
-    private String agreementId;
-    private String providerId;
-    private String reference;
-    private String returnUrl;
-    private MandateStatus state;
-    private MandateLinks links;
+    
+    private final String agreementId;
+    private final String providerId;
+    private final String reference;
+    private final String returnUrl;
+    private final String mandateReference;
+    private final MandateStatus state;
+    private final MandateLinks links;
 
     private GetMandateResponse(String agreementId,
                                String providerId,
                                String reference,
                                String returnUrl,
+                               String mandateReference, 
                                MandateStatus state,
                                MandateLinks links) {
         this.agreementId = agreementId;
         this.providerId = providerId;
         this.reference = reference;
         this.returnUrl = returnUrl;
+        this.mandateReference = mandateReference;
         this.state = state;
         this.links = links;
     }
 
     public static GetMandateResponse from(MandateConnectorResponse mandate, MandateLinks links) {
         return new GetMandateResponse(
                 mandate.getMandateId(),
-                mandate.getMandateReference(),
+                null,
```

Comment:
this is null at the moment as dd-connector doesn't return the provider id. this will be fixed in an upcoming PR.